internal/cli/install.go

```diff
@@ -514,6 +523,13 @@ func installRunner(
 	}
 	s.Done()
 
+	err = installutil.AdoptRunner(ctx, ui, client, id, advertiseAddr.Addr)
+	if err != nil {
+		s.Update("Error adopting runner: %s", err)
+		s.Status(terminal.StatusError)
+		return 1
```

Comment:
We typically do `c.ui.Output` instead of step groups on error returns, because step groups at the moment can be messy and it's not going to output the error as the "last" thing, but where the original step group was started. I recommend using `c.ui.Output` for any err handling. We also have a helper for humanizing errors:

https://github.com/hashicorp/waypoint/blob/f9b5e91a9ab0e9798f7c88428e6d1c36ef589155/internal/cli/status.go#L49-L56

This feedback goes for any where else too where we are returning errors in the CLI. Sorry I missed this feedback the first review around! ❤️